src/components/WorkflowForms/NewClientSignup/GatherInfo/hooks/useProviders.js

```
@@ -0,0 +1,269 @@
+    setEditingProvider(null);
+    setShowProviderModal(true);
+  };
+
+  useEffect(() => {
+    if (caseId) {
+      Promise.all([
+        loadProviderTypes(),
+        loadExistingProviders()
+      ]).then(() => {
+        setLoading(false);
+      }).catch(error => {
```

ESLint

[error] 'error' is defined but never used.

(no-unused-vars)